lib/agent0/agent0/hyperdrive/interactive/interactive_hyperdrive.py

```diff
@@ -130,6 +130,8 @@ class Config:
             The upper bound on the flat fee that governance can set.
         max_governance_fee: FixedPoint
             The upper bound on the governance fee that governance can set.
+        calc_pnl: bool
```

Comment:
I might be misremembering from before the holiday break, but I thought @slundqui and I said we did not want to add a `calc_pnl` flag here. iirc, the reasoning was that we don't want to establish a pattern that leads to bloat in this config (adding a flag for every modular change you want in the interface).

You are adding this flag because it is too slow when calculating PNL at every trade, yes? If so, I think the better solution is to remove the PNL calculation entirely as a default action, and then explicitly calculate a PNL column when it is needed and adding that column to the existing database (which I assume only happens when building the dashboard?)

Reply:
We decided that this flag isn't ideal, but the alternative solution is too difficult to handle right now. So we're going to leave the flag in and @wakamex will make an issue to remind us of the desired pattern.